lib/DownloadHandler.py

```diff
@@ -335,7 +336,8 @@ def download_site(self, url):
             self.logger.debug("Downloading from url: {}".format(url))
             session = self.get_session()
             try:
-                with session.get(url) as response:
+                with session.get(url, proxies={"http": conf.getProxy(),
```

Comment:
I would rather force the proxies into the current session; so the proxies should be set in the self.get_session() method. In that way there is a central place for all session related parameters.

Reply:
Fixed in 2270e15